src/pages/track-record.js

```diff
@@ -4,34 +4,78 @@ import {graphql} from 'gatsby';
+import * as sectionStyles from './track-record-section.module.scss';
 import ClientImages from '../components/client-images/client-images.js';
 
-const TrackRecordPage = ({data, location}) => {
-  /*
-   * To change the track record sections text change
-   * content/data/track-record-sections.yaml
-   * allTrackRecordSectionsYaml queries the file
-   * Query is at bottom of the page
-   */
-  const sections = data.allTrackRecordSectionsYaml.edges;
+const TrackRecordPage = ({location}) => {
   return (
     <Layout location={location}>
       <SEO title="Track Record"/>
       <h1 className={trackRecordStyles.title}>Track record</h1>
       <div className={trackRecordStyles.content}>
-        {sections.map(({key, node}) => {
-          return (
-            <TrackRecordSection
-              key={node.id}
-              trackRecord={node}
+        <div className={sectionStyles.container}>
+          <h2 className={sectionStyles.title}>
+            <img
+              className={sectionStyles.icon}
+              alt={`Generative AI Solutions icon`}
+              src={`/track-record/icon1.svg`}
             />
-          );
-        })}
+            Generative AI Solutions
+          </h2>
+          <p>Using LLMs and state-of-the-art techniques such as multi-agent systems and advanced prompt engineering to create generative AI solutions.</p>
+          <b>Previous projects:</b>
+          <ul>
+            <li><b>AiSentr:</b> delivered prototypes using Azure GPTs, RAG and multi-agentic framework to help close an important partnership for the business.</li>
+            <li><b>Automatic Classifiers for Peace (ACfP):</b> delivered an LLM-powered classifier to aid peacebuilders in finding insights from social media data. Open source repository and community, <a href="https://gitlab.com/automatic-classifiers-for-peace/acfp" target="_blank">here</a>.
+            </li>
+          </ul>
+        </div>
+        <div className={sectionStyles.container}>
+          <h2 className={sectionStyles.title}>
+            <img
+              className={sectionStyles.icon}
+              alt={`End-to-End AI Platforms icon`}
+              src={`/track-record/icon2.svg`}
+            />
+            End-to-End AI Platforms
+          </h2>
+          <p>Building and delivering a complete AI platform, that covers the full spectrum from data collection and processing to data visualization, derivation of insights and automated decision-making.</p>
+          <b>Previous projects:</b>
+          <ul>
+            <li><b>
+              <a
+                href="https://howtobuildup.org/phoenix/" target="_blank"
+              >
+                  Phoenix:
+              </a>
+            </b> a collaboration between <a href="https://howtobuildup.org/" target="_blank" >
+                Build Up
+              </a> and datavaluepeople, has helped 50+ peacebuilding organisations to draw insights and drive new initiatives based on social media analysis.</li>
```

Comment:
Think "draw insights and drive new initiatives based on social media analysis" is a bit weak. Can say something about large scale data driven evidence based initiative development" or something a bit more hard hitting.